src/workers/notifications/postAdded.ts

```diff
@@ -66,6 +74,18 @@ const worker: NotificationWorker = {
             } as NotificationPostContext & Partial<NotificationDoneByContext>,
           }),
         );
+
+        const posts = await con.getRepository(Post).countBy({
+          authorId: post.authorId,
+        });
```

Comment:
Don't think this one is complete.
We would need to check inside this squad scope as well right?

Reply:
Maybe we should also explicitly check for squad type post as they might be author of old articles?

Reply:
Though we check the Source Type to be Squad, I agree having to check the post type would be best.

Reply:
Yeah we do, but this would just get any posts count right? 👀
So the count could be different than what you want.

Reply:
Totally right!